src/Init/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+
+project(Samv71FreeRtos VERSION 1.0.0 LANGUAGES C)
```

Comment:
This should be renamed to something like `Samv71RuntimeInit`

Reply:
fixed in new commit